.rubocop_todo.yml

```diff
@@ -1,21 +1,29 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2017-07-17 15:31:02 -0700 using RuboCop version 0.49.1.
+# on 2017-09-14 20:48:51 -0700 using RuboCop version 0.50.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: Include, TreatCommentsAsGroupSeparators.
+# Include: **/Gemfile, **/gems.rb
+Bundler/OrderedGems:
```

Comment:
This seems unrelated to the scope of the PR.

Reply:
Yeah that was a mistake. Fixing